### PR TITLE
Content Validation STG 97

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Azure SDK for .NET
-
+Test
```
Comment (weshaggard) [inline README.md:2]:
```suggestion

```

**Comment by weshaggard** [comment]
@seanmcc-msft I push a comment and undid it just to refresh the merge branch which for whatever reason was missing and cause ci jobs to fail to start. The CI job is now moving.

**Comment by seanmcc-msft** [comment]
TODO update user-agent feature string to include Content Validation on this branch.  Merge, revert, and cherry-pick to 98.

**Pull request merged**